ingest-app/src/cmr/ingest/config.clj

```diff
@@ -38,7 +38,7 @@
 (defconfig variable-umm-version
   "Defines the latest variable umm version accepted by ingest - it's the latest official version.
    This environment variable needs to be manually set when newer UMM version becomes official"
-  {:default "1.8.2"})
```

Comment:
should we always use semantic versioning? 1.9.0?

Reply:
It looks like others on https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse/variable have two digits

Reply:
I have changed the units to 3: 1.9.0